independent-publisher-connectors/ITGlue/apiProperties.json

```diff
@@ -0,0 +1,23 @@
+{
+    "properties": {
+      "connectionParameters": {
+        "api_key": {
+          "type": "securestring",
+          "uiDefinition": {
+            "displayName": "API key",
+            "description": "The API key for ITGlue api",
+            "tooltip": "Provide your API key",
+            "constraints": {
+              "tabIndex": 2,
+              "clearText": false,
+              "required": "true"
+            }
+          }
+        }
+      },
+      "iconBrandColor": "#ED6100",
```

Comment:
@nk-gears Can you change it default iconBrandColor for all the IP connectors?

`"iconBrandColor": "#da3b01"`

Reply:
@sriyen-msft Thank you for the feedback. I have fixed the description. Also i fixed the branding color. Regarding the color i was looking at Etsy IP and followed the same color. May be Etsy also needs an update.

BTW, Any update on my another IP Connector FreeAgent as i have not received from that thread.

Reply:
@nk-gears There is an issue in the pipeline which was delayed your connector's deployment and it was resolved today. We will queue up your connector for tomorrow's train which will be completed by Thursday 7th October 2021. Thanks